### DISCUSSION TITLE
Can't access dashboard of su-abase

### DISCUSSION THREAD

**Discussion by faabalia** [supabase/supabase#2574] [Questions] [answered]
Hi,
I registered in supabase with my GitHub user but when I try to access now I always go to this page:
<img width="1440" alt="Captura de pantalla 2021-07-26 a las 15 28 07" src="https://user-images.githubusercontent.com/3009220/127010272-52a94471-e072-45e2-8c59-2755dfd7e099.png">
If I do Log in or sign in I go back to this page. I have tries to access in Safari and Chrome. I am working in a MacBook Pro with chip M1 with macOS Monterrey beta 3

The first time when I registered I was working fine and I created a project with tables and other elements without any problem, but in the next access is when I had the problem that I say.

Thanks in advance and greeting,

Felix.

**Comment by 72L** [comment, marked as answer]
There's an issue filed here: #2568

~~For immediate relief, you can try using a VPN. Singapore works for me!~~
Singapore is no longer working
